src/components/Alert/Alert.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
-
+import { ThemeProvider } from 'styled-components';
 import { StyledAlert } from './Alert.styled';
 
-const Alert = props => {
+const theme = {
+    default: {
+        color: 'red',
+        backgroundColor: 'white',
+        borderColor: 'red',
+    },
+    primary: {
+        color: '#004085',
+        backgroundColor: '#cce5ff',
+        borderColor: '#b8daff',
+    },
+    secondary: {
+        color: '#383d41',
+        backgroundColor: '#e2e3e5',
+        borderColor: '#d6d8db'
+    }
+}
+
+
+const Alert = (props) => {
+    const {variant = "default"} = props
     return (
-        <StyledAlert>{props.children}</StyledAlert>
+        <ThemeProvider theme={theme[variant]}>
```

Comment:
Raczej ten element wrzucamy poza komponent bo chcemy, aby wszystkie elementy miały podobny motyw tz. jeśli chcemy go zmieniać (motyw) to zmieniamy na całej stronie, a nie w jednym miejscu.